### PR TITLE
Task sf/j13 backend create home page

### PR DESCRIPTION
**Pull request by Sayonarko**

### CODE REVIEW

File: shared/src/common/types/podcast/podcast.type.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+type Podcast = {
+  id: number;
+  name: string;
+  user_id: number;
```
Comment (ghost) [inline shared/src/common/types/podcast/podcast.type.ts]:
```suggestion
  userId: number;
```

File: backend/src/services/podcast/podcast.service.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  constructor({ podcastRepository }: Constructor) {
+    this.#podcastRepository = podcastRepository;
+  }
+
+  public getAll(): Promise<TPodcast[]> {
+    return this.#podcastRepository.getAll();
+  }
+
+  public create(payload: TPodcast): Promise<TPodcast> {
```
Comment (ghost) [inline backend/src/services/podcast/podcast.service.ts]:
```suggestion
  public create(payload: PodcastCreatePayload): Promise<TPodcast> {
```

File: backend/src/api/podcasts/podcasts.api.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    PodcastsApiPath.ROOT,
+    handleAsyncApi(async (_req, res) => {
+      return res.json(await podcastService.getAll()).status(HttpCode.OK);
+    }),
+  );
+
+  podcastRouter.post(
+    PodcastsApiPath.ROOT,
+    handleAsyncApi(async (_req, res) => {
+      return res.json(await podcastService.create(_req.body)).status(HttpCode.OK);
```
Comment (ghost) [inline backend/src/api/podcasts/podcasts.api.ts]:
```suggestion
      return res.json(await podcastService.create(_req.body)).status(HttpCode.CREATED);
```

Comment (ghost) [inline backend/src/api/podcasts/podcasts.api.ts]:
```suggestion
    handleAsyncApi(async (req, res) => {
      return res.json(await podcastService.create(req.body)).status(HttpCode.OK);
```
`_` only for unused parametres

**Review by ghost — CHANGES_REQUESTED** [review]

File: backend/src/common/enums/db/podcast-dto-key.enum.ts [hunk 4]
```diff
@@ -0,0 +1,6 @@
+enum PodcastDTOKey {
+  NAME = 'name',
+  USER_ID = 'user_id',
```
Comment (ghost) [inline backend/src/common/enums/db/podcast-dto-key.enum.ts]:
```suggestion
  USER_ID = 'userId',
```

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Review by yevhenprybytok — DISMISSED** [review]

File: shared/src/common/enums/podcast/podcast-create-payload-key.enum.ts [hunk 5]
```diff
@@ -0,0 +1,6 @@
+enum PodcastCreatePayloadKey {
+  NAME = 'name',
+  USER_ID = 'user_id',
```
Comment (ghost) [inline shared/src/common/enums/podcast/podcast-create-payload-key.enum.ts]:
```suggestion
  USER_ID = 'userId',
```

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]
👍

**Pull request merged**